packages/lesswrong/components/widgets/Paper.tsx

```diff
@@ -51,6 +52,10 @@ export const Paper = ({elevation=2, square=false, className, style, children}: {
 const cardStyles = defineStyles("Card", (theme) => ({
   root: {
     overflow: "hidden",
+    ...(isLW && {
+      borderRadius: 1,
+      boxShadow: theme.palette.boxShadow.lwCard,
+    })
```

Comment:
@jimrandomh Before/after of this change, it's pretty subtle:
![Screenshot 2025-04-17 at 14 07 10](https://github.com/user-attachments/assets/5e277493-d00e-416e-8851-552b3717f74c)

![Screenshot 2025-04-17 at 14 04 24](https://github.com/user-attachments/assets/d2fec71b-c3c7-492b-827c-543afcdf557d)